internal/trie/node/decode.go

```diff
@@ -101,11 +108,10 @@ func decodeBranch(reader io.Reader, variant byte, partialKeyLength uint16) (
 				ErrDecodeChildHash, i, err)
 		}
 
-		const hashLength = 32
 		childNode := &Node{
 			MerkleValue: hash,
 		}
-		if len(hash) < hashLength {
+		if len(hash) < common.HashLength {
```

Comment:
can we define an internal `hashLength` constant local to this package?  It's kinda ugly having references to `common` everywhere.